### ISSUE TITLE
Scrolling list of instructions with mouse wheel is broken

### ISSUE THREAD

**Issue by daniel-j-h** [Project-OSRM/osrm-frontend-v2#195]
Mouse wheel over the map should zoom in / out. (works)
Mouse wheel over the instruction list should scroll the list up / down. (broken)

cc @TheMarex

**Comment by perliedman** [comment]
Hm, this works for me (Ubuntu 16.04, Chrome 54.0.2840.59).
- What happens when you wheel scroll? Nothing / map zooms / ...?
- Is it always broken for you, or any particular route, etc.?
- Does it work for you in the basic LRM demo (http://www.liedman.net/leaflet-routing-machine/)?
- What system/browser?

**Comment by daniel-j-h** [comment]
Yep Chrome works fine. Seems to be Firefox having issues. Ubuntu 15.04.

LRM demo works fine in Firefox - looks like demo server specific problem.

**Comment by donno** [comment]
This issue was recently reported to Mozilla two months ago as [Bug 1942589](https://bugzilla.mozilla.org/show_bug.cgi?id=1942589).. Their own investigation came to the conclusion that the problem is not in osrm-frontend or leaflet but is specific to the demo server.

One of the parties involved [notified ](https://lists.openstreetmap.org/pipermail/osrm-talk/2025-January/002100.html) the osrm-talk mailing list, so I figured it would be worth forwarding it along here.

I tested it with Firefox 136.0 (64-bit) and Microsoft Edge Version 134.0.3124.66 (Official build) (64-bit),(which is essentially Chrome these days) and Firefox still suffered the problem.

The project `fossgis-routing-server/osrm-frontend` doesn't have the issue tracker turned on to report the issue there.

**Comment by DennisOSRM** [comment]
This works now:

<img width="1279" height="1037" alt="Image" src="https://github.com/user-attachments/assets/a3d156ba-3c70-4cad-978f-78b8473e7017" />
